TCAT/Controllers/ParentHomeViewController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ParentHomeViewController.swift
+//  TCAT
+//
+//  Created by Lucy Xu on 11/4/19.
+//  Copyright © 2019 cuappdev. All rights reserved.
+//
+
+import UIKit
+import Pulley
+
+class ParentHomeMapViewController: PulleyViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    required init(contentViewController: UIViewController, drawerViewController: UIViewController) {
+        super.init(contentViewController: contentViewController, drawerViewController: drawerViewController)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
```

Comment:
doesn't seem like this is necessary unless we plan on adding stuff later?

Reply:
The initializations are necessary for Pulley!